k8s/demo/pvc-sparse-claim-cstor.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
See if these can be added into k8s/sample-yamls and also update the maya-apiserver ci to make use of the new location.

Reply:
Done!